packages/lib/README.md

```diff
@@ -13,7 +13,7 @@ The blueprintsManager works with an api provided by the user (see blow)
 
 | Function Name | description       |  arguments  | return
 | --------------- | ----------- | ---------- | ---------|
-| async `createBlueprint` | Used when changes were made that might require a new blueprint, params can be empty - the manager will fetch by the provided api | params: {styles, items, container, totalItemsCount} | {blueprint: object, blueprintChange: boolean}
+| async `createBlueprint` | Used when changes were made that might require a new blueprint, params can be empty - the manager will fetch by the provided api | params: {styles, items, dimensions, totalItemsCount} | {blueprint: object, blueprintChange: boolean}
```

Comment:
This readme should be removed since we have types for documentation now. No need to sync docs and code with public API anymore

Reply:
I suggest keeping it until Yonatan and you will remove those files :)

Reply:
We can remove these files once we are done (with all our documentation in mind). Rita you can add a task for this on the board.